build.sh

```diff
@@ -54,19 +48,38 @@ function generate_image() {
         memdisk biosdisk part_msdos part_gpt gfxterm_menu fat tar bsd memrw configfile
 }
 
-if [ "$1" == "clean" ]; then
-  cd ${path_orbital}
-  cd ${path_bios}
-  make clean
-  cd ${path_orbital}
-  cd ${path_grub}
-  make clean
+while [ $# -ne 0 ]
+do
+	arg="$1"
```

Comment:
Indentation in this file uses two spaces, you are using tabs. Can you fix this?

Reply:
Yeah, sure. My IDE automatically does tabs.

Reply:
Same goes for the lines below.